snf/wrapper.h

```diff
@@ -265,7 +265,9 @@ static inline struct compound_int snf_inject_send_bulk(snf_inject_t inj,
 				int timeout_ms, int flags, uintptr_t *pkts, uint32_t n_pkts,
 				const uint32_t *lengths)
 {
-	return ENOTSUP;
+	struct compound_int out;
+	out.rc = ENOTSUP
```

Comment:
the other  stuff in `out` in unset, i.e. is set to gibberish

Reply:
I just removed mock function